groupthink/commands.py

```diff
@@ -2,107 +2,129 @@
+        return 'Could not get {0}-cli command. Please make sure https://github.com/{0}/{0}-cli exists.'.format(group)
 
 
-def uninstall(org, script_dir=script_dir, repo_dir=repo_dir):
-    # remove organization scripts repo and groupthink script
-    check_install(org, check_already=True, repo_dir=repo_dir)
-    execute_cmd(['rm', '-rf', '{0}/{1}-cli'.format(repo_dir, org)])
-    execute_cmd(['rm', '{0}/{1}'.format(script_dir, org)])
-    return 'Removed {} command.'.format(org)
+@arg('group', help='The group to uninstall')
+@arg('-d','--dest')
+@arg('-s', '--storage')
+def uninstall(group, dest=dest, storage=storage):
+    # remove group scripts repo and groupthink script
+    check_install(group, check_already=True, storage=storage)
+    execute_cmd(['rm', '-rf', '{0}/{1}-cli'.format(storage, group)])
+    execute_cmd(['rm', '{0}/{1}'.format(dest, group)])
+    return 'Removed {} command.'.format(group)
 
 
-def update(org, script_dir=script_dir, repo_dir=repo_dir):
+@arg('group', help='The group to update')
+@arg('-d','--dest')
+@arg('-s', '--storage')
+def update(group, dest=dest, storage=storage):
     # check if there are any updates to the scripts repository
-    check_install(org, check_already=True, repo_dir=repo_dir)
-    update_cmd = ['git', '--git-dir={0}/{1}-cli/.git'.format(repo_dir, org), 'fetch']
+    check_install(group, check_already=True, storage=storage)
+    update_cmd = ['git', '--git-dir={0}/{1}-cli/.git'.format(storage, group), 'fetch']
     (msg, err) = execute_cmd(update_cmd)
     if msg:
-        return "There are updates to the {0} command. Use the --upgrade subcommand to add them".format(org)
+        return 'There are updates to the {0} command. Use the upgrade subcommand to add them'.format(group)
     else:
-        return "Your {0} command is already up to date.".format(org)
+        return 'Your {0} command is already up to date.'.format(group)
+
 
+@arg('group', help='The group to upgrade')
+@arg('-d','--dest')
+@arg('-s', '--storage')
+def upgrade(group, dest=dest, storage=storage):
+    # upgrade group scripts repository
 
-def upgrade(org, script_dir=script_dir, repo_dir=repo_dir):
-    # upgrade organization scripts repository
-    check_install(org, check_already=True, repo_dir=repo_dir)
-    update_cmd = ['git', '--git-dir={0}/{1}-cli/.git'.format(repo_dir, org), 'pull']
+    check_install(group, check_already=True, storage=storage)
+    update_cmd = ['git', '--git-dir={0}/{1}-cli/.git'.format(storage, group), 'pull']
     (msg, err) = execute_cmd(update_cmd)
     if msg.find('Already up-to-date') > -1:
-        return "Your {0} command is already up to date.".format(org)
+        return 'Your {0} command is already up to date.'.format(group)
     else:
-        return 'Upgraded {} command.'.format(org)
+        return 'Upgraded {} command.'.format(group)
 
 
-def list_orgs(repo_dir=repo_dir):
-    # print out a list of organizations whose scripts have been
+@aliases('list')
```

Comment:
I'd rather not, since that would overwrite it and make the command available only as `list`. If you'd prefer that `list` be the name, I'll do that in `parser.add_commands()`.